app/javascript/components/recipes/new/list.vue

```diff
@@ -135,6 +137,11 @@ export default {
       this.steps[index] = newText;
       this.$forceUpdate();
     },
+    delIngredient(ingredient) {
+      const index = this.ingredients.indexOf(ingredient);
+      this.ingredients.splice(index, 1);
+      this.$forceUpdate();
+    },
```

Comment:
Lo mejor acá es reemplazar completamente el arreglo en lugar de forzar el update, tratamos de no hacerlo. Yo generalmente lo hago a través de un `filter`, así:

```suggestion
    delIngredient(ingredient) {
      this.ingredients = this.ingredients.filter((originalIngredient) => originalIngredient.id !== ingredient.id);
    },
```

Arriba también veo que se usa harto 😢  trataría de hacer un refactor en un futuro. Cuéntame qué te parece

Reply:
Entiendo perfecto! No creo que se use taanto, asi que deberia salir bien rapido el refactor! :) graciass